charts/polymorphic-app/values.yaml

```diff
@@ -114,6 +114,19 @@ serviceTemplate:
     #    hosts:
     #      - chart-example.local
 
+  httpRoute:
```

Comment:
We could add a comment here that this is supposed to replace the ingress functionality.